src/vsr/superblock.zig

```diff
@@ -52,6 +52,7 @@ pub const SuperBlockSector = extern struct {
     size: u64,
 
     /// The maximum size of the data file.
+    // TODO Actually limit the file to this size.
```

Comment:
Indeed. I also meant this to be here as a TODO! It's our "cgroups" for data files.